src/generic/gencli.tcl

```diff
@@ -1051,6 +1055,13 @@ proc keepalive {} {
 	if {$bm eq "TPC-C"} {
 #For TPC-C we have a rampup and duration time, find these, check they are valid and call _runtimer automatically with these values
 	 upvar #0 dbdict dbdict
+	 upvar #0 genericdict genericdict
+         if {[dict exists $genericdict commandline keepalive_margin]} {
+                set ka_margin [ dict get $genericdict commandline keepalive_margin]
+                if {![string is entier $ka_margin]} { set ka_margin 10 }
```

Comment:
The if and else branches do the same thing - do we need the condition at all?

Reply:
I understand now.  I had misread the code.

I thought the else on line 1062 was for the if on line 1061, but it's actually for the if on line 1059.  

Can we fix the indentation on lines 1062-1064 so that it is aligned properly with line 1059?

Reply:
I have updated the indentation. There is a package included with HammerDB called reformat_tcl. You can do "package require reformat_tcl" and then reformat_src indent filename, and it will update the file with the correct indentation. This should server as a prompt for me to use it to prevent less confusion going forward.